api/envoy/api/v2/core/base.proto

```diff
@@ -76,10 +77,45 @@ message Locality {
   string sub_zone = 3;
 }
 
+// BuildVersion combines SemVer version of extension with free-form build information
+// (i.e. 'alpha', 'private-build') as a set of strings.
+message BuildVersion {
+  // SemVer version of extension
+  type.SemanticVersion version = 1;
+
+  // Free-form build information i.e. 'alpha'
+  repeated string labels = 2;
+}
+
+// Version and identification for an Envoy extension.
+// [#next-free-field: 6]
+message Extension {
+  // This is the name of the Envoy filter as specified in the Envoy
+  // configuration, e.g. envoy.router, com.acme.widget.
+  string name = 1;
+
+  // Category of the extension.
+  // Extension category is provided by the `category()` method of the factory class.
+  // TODO(yanavlasov): Link to the doc with existing category names.
```

Comment:
nit: please make this a comment TODO that won't render in the docs. Same below.

Reply:
done